float32 klst  # kLST linear stiffness
float32 kast  # kAST angular stiffness
float32 kvst  # kVST volume stiffness
uint8 bending_distance
int32 flags  # Default?
